kubernetes/resource_kubernetes_config_map_test.go

```diff
@@ -345,3 +377,40 @@ resource "kubernetes_config_map" "test" {
 }
 `, prefix)
 }
+
+func testAccKubernetesConfigMapConfig_binaryData(prefix string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_config_map" "test" {
+  metadata {
+    generate_name = "%s"
+  }
+
+  binary_data {
+    one = "${file("./test-fixtures/binary.data")}"
```

Comment:
I'm afraid this approach will be very short lived.
Starting with Terraform 0.12 it will be illegal to pass non-UTF8 data around in configuration or reading it from files via the `file()` function.

Documented [here](https://github.com/hashicorp/terraform/blob/26c1e40ad775297cc89b32794508f4d596615d5a/configs/configupgrade/upgrade_expr.go#L348-L350)